src/index.html

```diff
@@ -6,9 +6,32 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="/">
+        <img class="logo_image" src="images/logo.png" 
+             alt="Nav logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item"><a href="#" class="nav_link is-active">Apple</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Samsung</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Smartphones</a></li>
+          <li class="nav_item">
+            <a href="#" class="nav_link" 
+            data-qa="hover" >Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a 
              href="#" 
              class="nav_link" 
              data-qa="hover" 
            >
              Laptops & Computers
            </a>
```

Reply:
if you have more than 2 attributes - move every to new line with 2 spaces indentation